hbase-server/src/main/java/org/apache/hadoop/hbase/replication/master/ReplicationHFileCleaner.java

```diff
@@ -79,6 +78,9 @@ public boolean apply(FileStatus file) {
         }
         return !foundHFileRefInQueue;
       }
+      public boolean test(FileStatus file) {
```

Comment:
Every use of Predicate gets this update. It is annoying. Guava's predicate extends Java's Predicate. At some point between Java 7 and Java 11 (which I am using as primary compiler these days) this method 'test' is now required by the Java Predicate interface, even if it is only apply() that is the interesting method that will be called. To preserve compatibility with earlier versions no Override annotation is placed here. 

Perhaps long term we should migrate away from use of Guava's Predicate. 

I will include this comment once per compilation unit where this change was necessary.